packages/appkit/exports/react.ts

```diff
@@ -2,6 +2,7 @@ import { CoreHelperUtil, type UseAppKitNetworkReturn } from '@reown/appkit-contr
 import { useAppKitNetworkCore } from '@reown/appkit-controllers/react'
 import type { AppKitNetwork } from '@reown/appkit/networks'
 
+import type { AppKitBaseClient, OpenOptions, Views } from '../src/client/appkit-base-client.js'
```

Comment:
Wouldn't the following works?

```suggestion
export type { AppKitBaseClient, OpenOptions, Views } from '../src/client/appkit-base-client.js'
```

Reply:
Yes, it works. I'll commit these changes